src/main/kotlin/services/userService.kt

```diff
@@ -0,0 +1,60 @@
+package services
+
+import com.google.gson.GsonBuilder
+import models.UserModel
+import org.json.JSONObject
+import repo.UserRepo
+import java.time.LocalDateTime
+import java.util.*
+
+class Service {
+
+    fun handleGetAll(limit: Int, offset: Int, json: String): MutableList<UserModel> {
+        return UserRepo().getAllUsers(limit, offset, json)
+    }
+
+    fun handleGet(id: String, json: String): UserModel? {
+        return UserRepo().getUserById(id, json)
+    }
+
+    fun handlePost(json: String): UserModel? {
+        val gson = GsonBuilder().create()
+        val user = gson.fromJson(json, UserModel::class.java) //map json to userModel
+        user.timeCreated = LocalDateTime.now().toString()
```

Comment:
use time as System.currentTimeMillis, it will give epoch value(Long)
which will be useful in every case, make timeCreated as of type Long